RELEASE_NOTES.md

```diff
@@ -27,6 +27,17 @@
 [`int8` or `int4` compressed key and value]: https://oneapi-src.github.io/oneDNN/v3.7/dev_guide_graph_sdpa_compressed_kv.html#doxid-dev-guide-graph-sdpa-compressed-kv
 [Gated Multi-Layer Perceptron (Gated MLP)]: https://oneapi-src.github.io/oneDNN/v3.7/dev_guide_graph_gated_mlp.html#doxid-dev-guide-graph-gated-mlp
 
+## AArch64-based Processors
+  * Add support for the Arm Compute Library (ACL) thread_local scheduler via ThreadpoolScheduler
```

Comment:
This one belongs in features or usability section, as it sounds like new capability.